test/Microsoft.AspNetCore.Server.HttpSys.FunctionalTests/AuthenticationTests.cs

```diff
@@ -294,14 +233,13 @@ public class AuthenticationTests
         [FrameworkSkipCondition(RuntimeFrameworks.CoreCLR, SkipReason = "HttpClientHandler issue (https://github.com/dotnet/corefx/issues/5045).")]
         public async Task AuthTypes_ChallengeWithoutAuthTypes_AllChallengesSent(AuthenticationSchemes authType)
         {
-            string address;
             var authTypeList = authType.ToString().Split(new char[] { ',', ' ' }, StringSplitOptions.RemoveEmptyEntries);
-            using (Utilities.CreateHttpAuthServer(authType, AllowAnoymous, out address, httpContext =>
+            using (var server = Utilities.CreateDynamicHost(string.Empty, authType, AllowAnoymous, out var address, out var baseAddress, httpContext =>
             {
                 Assert.NotNull(httpContext.User);
                 Assert.NotNull(httpContext.User.Identity);
                 Assert.False(httpContext.User.Identity.IsAuthenticated);
-                return httpContext.Authentication.ChallengeAsync();
+                return httpContext.ChallengeAsync(authType.ToString());
```

Comment:
How can it do multiple itself? Make one handler for 'Windows' and have it issue challenges for all enabled schemes?